docs/plugins/repository.asciidoc

```diff
@@ -8,37 +8,20 @@ by distributed file systems:
 [discrete]
 ==== Core repository plugins
 
-The core repository plugins are:
-
-<<repository-s3,S3 Repository>>::
-
-The S3 repository plugin adds support for using S3 as a repository.
+NOTE: The S3, GCS and Azure repository plugins are now included in the {es}
```

Comment:
Since they're no longer "plugins" I'd reword this to say

> NOTE: Support for S3, GCS and Azure repository is now bundled in {es} by default

I'd also rename the section "Core repository plugins" to "Official repository plugins".

@elastic/es-docs thoughts?